StorageUtility/Linux/StorageUtility.cpp

```diff
@@ -66,10 +71,9 @@ bool IsAtaDeviceBus(sStorageAdapterProperty StorageAdapterProperty)
     return(StorageAdapterProperty.AtaBus);
 }
 
-bool IsScsiDeviceBus(sStorageAdapterProperty StorageDeviceProperty)
+bool IsScsiDeviceBus(sStorageAdapterProperty StorageAdapterProperty)
```

Comment:
Hi a.Tam, It same code of VTStor-Window, if change ref bypass then we have a error build code